src/adapters/moziot-adapter-example/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "moziot-adapter-example",
+  "version": "0.2.1",
+  "description": "Example adapter plugin for Mozilla IoT Gateway",
+  "main": "index.js",
+  "keywords": [
+    "moziot",
+    "adapter",
+    "example"
+  ],
+  "homepage": "https://iot.mozilla.org/",
+  "license": "MPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mozilla-iot/gateway.git"
+  },
+  "bugs": {
+    "url": "https://github.com/mozilla-iot/gateway/issues"
+  },
+  "moziot": {
+    "api": {
+      "min": 1,
+      "max": 1
+    },
+    "plugin": true
```

Comment:
What adapters are not plugins?

Reply:
Eventually, this can be removed. I put it there so that we could have the option of having adapters which were loaded with the gateway (as opposed to being in a separate process)